hack/dockerfile/install/proxy.installer

```diff
@@ -3,7 +3,7 @@
 # LIBNETWORK_COMMIT is used to build the docker-userland-proxy binary. When
 # updating the binary version, consider updating github.com/docker/libnetwork
 # in vendor.conf accordingly
-LIBNETWORK_COMMIT=ed2130d117c11c542327b4d5216a5db36770bc65
+LIBNETWORK_COMMIT=2112cace8efd4c7b71daccbf54897871e205f293
```

Comment:
updating userland-proxy accordingly, but it was not in sync with the vendor https://github.com/docker/libnetwork/compare/ed2130d117c11c542327b4d5216a5db36770bc65... 2112cace8efd4c7b71daccbf54897871e205f293 (don't think this brings in any changes in the proxy)